packages/dataviews/src/types.ts

```diff
@@ -463,6 +463,11 @@ export interface ActionModal< Item > extends ActionBase< Item > {
 	 * The header of the modal.
 	 */
 	modalHeader?: string;
+
+	/**
+	 * The size of the modal.
```

Comment:
```suggestion
	 * The size of the modal.
	 *
	 * @default 'medium'
```

Unlike the `Modal` component, the `ActionsModal` component has the default size value, so it is better to specify it.

Also, let's update the README: https://github.com/WordPress/gutenberg/blob/trunk/packages/dataviews/README.md#modalheader